geometry/TrianglesView.swift

```diff
@@ -36,4 +38,19 @@ class TrianglesView: UIView {
     func setDataSource(dataSource: TrianglesProtocol) {
         self.dataSource = dataSource
     }
+    func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        let settingsViewController = segue.destinationViewController as! SettingsViewController
+        settingsViewController.delegate = self
+        settingsViewController.red = redt
+        settingsViewController.green = greent
+        settingsViewController.blue = bluet
+    }
+}
+
+extension TrianglesView: SettingsViewControllerDelegate {
```

Comment:
extension would be useful to add new functionality to a library class. In this simple case TrianglesView can implement SettingsViewControllerDelegate: `class TrianglesView: UIView, SettingsViewControllerDelegate {`